server/src/lib.rs

```
@@ -808,7 +812,13 @@ where
             .db(&name)
             .context(DatabaseNotFound { db_name: &db_name })?;
 
-        Ok(db.load_chunk_to_read_buffer_in_background(table_name, partition_key, chunk_id))
+        let chunk = db
+            .lockable_chunk(&table_name, &partition_key, chunk_id)
+            .context(ChunkNotFound)?;
```

This is a side-benefit of the new Db API - previously the gRPC API would fail silently if you asked to manipulate a non-existent chunk